controls/cusp_fedora.yml

```diff
@@ -0,0 +1,517 @@
+---
+policy: 'Fedora Common User Security Policy'
+title: 'Fedora Common User Security Policy'
+id: cusp_fedora
+version: '1.0.0'
+source: "jodehnal's bachelor thesis on creating a SCAP profile for common users of Fedora workstation - link will be added after publication"
+
+controls:
+  ###
+  ### 1. Hardware and its configuration
+  ###
+  - id: 1.1
+    title: Protection of the BIOS or UEFI
+    description: >-
+      Users should protect their BIOS or UEFI with a password.
+    status: manual
+
+  - id: 1.2
+    title: Proper BIOS or UEFI configuration
+    description: >-
+      Users should disable features and devices in the BIOS or UEFI that are not in use and should only include trusted devices in the boot order.
+    status: manual
+
+  - id: 1.3
+    title: 64-bit OS
+    description: >-
+      When possible, users should use a 64-bit system and hardware that supports it.
+    status: manual
+  ###
+  ### 2. System installation
+  ###
+  - id: 2.1
+    title: Security policy selection
+    description: >-
+      Users should apply the “Fedora Common User Security Policy” in the installer.
+    status: manual
+
+  - id: 2.2
+    title: Disk partitioning
+    description: >-
+      Users should put the /home, /tmp, /var, /var/tmp and /var/log directories on separate partitions.
+    status: manual
```

Comment:
The rules from `linux_os/guide/system/software/disk_partitioning` might be useful here

Reply:
Yes, the issue is that the respective Anaconda remediations block the installation when the disk is not correctly partitioned which is not ideal, I talk about this in the thesis and created an issue in OAA https://github.com/OpenSCAP/oscap-anaconda-addon/issues/236